EIPS/eip-3722.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: 3722
+title: Poster
+description: A ridiculously simple general purpose social media smart contract.
+author: Auryn Macmillan (@auryn-macmillan)
+discussions-to: (https://ethereum-magicians.org/t/eip-poster-a-ridiculously-simple-general-purpose-social-media-smart-contract/6751)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-31
+---
+
+# Poster
+
+## Abstract
+A ridiculously simple general purpose social media smart contract.
+It takes a string as a parameter and emits that string, along with msg.sender, as an event. That's it.
+The EIP also includes a proposed standard json format for a twitter-like application, where each `post()` call can include multiple posts and/or operations. The assumption being that application state will be constructed off-chain via some indexer.
+
+## Motivation
+Poster is intended to be used as a base layer for decentralized social media. It can be deployed to the same address (via the singleton factory) on just about any EVM compatible network. Any Ethereum account can make posts to the deployement of Poster on its local network.
```

Comment:
```suggestion
Poster is intended to be used as a base layer for decentralized social media. It can be deployed to the same address (via the singleton factory) on just about any EVM compatible network. Any Ethereum account can make posts to the deployment of Poster on its local network.
```